recipe/meta.yaml

```
@@ -150,17 +179,30 @@ outputs:
         - libfaiss{{ libext }} =*=*_{{ faiss_proc_type }}
     requirements:
       build:
+        - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-        # necessary for conda-build to build the python bindings per version
-        - python                  # [build_platform != target_platform]
         - cmake
         - ninja
+        # libgomp is required for linux-64 alongside intel-omp.
         - libgomp                 # [linux]
-        - llvm-openmp             # [osx]
+        # llvm-openmp is required only for osx
+        - llvm-openmp  14.0.6     # [osx]
       host:
-        - libblas
-        - liblapack
+        - python
+        # OpenBLAS + libgomp for osx-arm64, and linux-s390x and linux-ppc64le, linux-aarch64.
+        - openblas 0.3.21         # [s390x or ppc64le or aarch64 or arm64]
```

```suggestion
        - openblas {{ openblas }}         # [s390x or ppc64le or aarch64 or arm64]
```
Because it's inside cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/346f42e1e793e3617247734a7789ab3238271b2c/conda_build_config.yaml#L163

---

I tried that and I now remember why we can't do that. I think there is a conda bug. With such specification, it thinks the input is a dictionary. I think it is because openblas and mkl are part of the blas_impl matrix/list/array. 

The error:
`Received dictionary as spec.  Note that pip requirements are not supported in conda-build meta.yaml.  Error message: Invalid version '': empty version string`